operator-framework-core/src/test/java/io/javaoperatorsdk/operator/processing/DefaultEventHandlerTest.java

```diff
@@ -224,6 +232,14 @@ public void scheduleTimedEventIfInstructedByPostExecutionControl() {
         .scheduleOnce(any(), eq(testDelay));
   }
 
+  @Test
+  public void doNotFireEventsIfClosing() {
+    defaultEventHandler.close();
+    defaultEventHandler.handleEvent(prepareCREvent());
+
+    verify(eventDispatcherMock, timeout(50).times(1)).handleExecution(any());
```

Comment:
Shouldn't we not dispatch at all if the handler is closed?

Reply:
mh yes, don't know why I ended up with this test

Reply:
will fix it later today

Reply:
@metacosm fixed